buildpipeline/DotNet-CoreFx-Trusted-Windows.json

```diff
@@ -424,6 +424,9 @@
     },
     "PB_CleanAgent": {
       "value": "true"
+    },
+    "PB_SymbolsBuildId": {
+      "value": "DotNet-CoreFx-Windows"
```

Comment:
I think this would cause collisions if two pipebuilds use the same checked-in definition, since the build numbers would be independent. I don't know if a collision causes any harm, though. To be safe, maybe `DotNet-CoreFx-Windows-$(System.DefinitionId)`.